jsk_naoqi_robot/naoeus/README.md

```diff
@@ -8,4 +8,11 @@ Install nao mesh files from deb with manual approval of license
 ```
 sudo apt-get install ros-<ros version>-nao-meshes 
 catkin build
-```
\ No newline at end of file
+```
+
+
+How to start jsk_nao_startup
+-----------------------------
+```
+roslaunch jsk_nao_startup.launch network_interface:=enp0s25
```

Comment:
put package name and format as `roslaunch <package_name> <launch_filename> <args>`